### PR TITLE
DSG-2818: Side bar hover tooltips

### PR DESCRIPTION
**Pull request by sunnyjiao-nv**
# Basics

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?
- Adds a tooltip on hover for the sidebar icons in Employer portal

## Screenshot of the affected changes

https://github.com/Nuvalence/dsgov-web/assets/155997897/b5086c75-dbcd-4669-9630-96be8bed7cac

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/shared/ui/nuverial/src/lib/components/side-bar-menu/side-nav-menu.component.html [hunk 1]
```diff
@@ -3,17 +3,19 @@
...
         </ng-container>
 
         <ng-container *ngFor="let navMenuItem of navBarMenuItemList; trackBy: trackByFn">
           <mat-list-item
+            [matTooltip]="navMenuItem.label || ''"
             [queryParams]="navMenuItem.navigationParams"
             [routerLink]="[navMenuItem.navigationRoute]"
             (click)="onMenuClick(navMenuItem)"
             mat-list-item
+            matTooltipPosition="right"
```
Comment (sunnyjiao-nv) [inline libs/shared/ui/nuverial/src/lib/components/side-bar-menu/side-nav-menu.component.html:18]: Do want want to make the tooltip position configurable?

**Review by kevincheng0 — APPROVED** [review]
Looks good to me

File: libs/shared/ui/nuverial/src/lib/components/side-bar-menu/side-nav-menu.component.html (hunk 1 above)
Comment (davidyoung77) [reply libs/shared/ui/nuverial/src/lib/components/side-bar-menu/side-nav-menu.component.html:18]: Sure you can use an optional input with a default value

**Review by davidyoung77 — APPROVED** [review]
These changes all look good.

**Review by davidyoung77 — APPROVED** [review]

**Pull request merged**